lib/middleware/redirects.js

```diff
@@ -54,15 +61,15 @@ Redirect.prototype.test = function(url) {
 
       return {
         type: this.type,
-        destination: dest
+        destination: dest + qs
```

Comment:
I think it's possible for destinations to contain a query string - do we want to check and possibly merge query strings (overwriting? adding? if we get a collision)